<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_view</class>
 <widget class="QWidget" name="Admin_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>594</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>985</width>
    <height>594</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>985</width>
    <height>594</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="current_profile_pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../noprofile1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_selected_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 8pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_log_out">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 8pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 8pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit_user">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 8pt &quot;Microsoft JhengHei UI Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 170, 127);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_selected_tel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 8pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Tel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_selected_province">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 8pt &quot;Microsoft JhengHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Province</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_all_user">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>109</y>
     <width>971</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 127);</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>969</width>
      <height>479</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
